website/blog/dec-2021/structure-chaining.md

```diff
@@ -0,0 +1,69 @@
+---
+{
+"TableOfContents": {
+"Name": "Silk.NET Structure Chaining",
+"Url": "structure-chaining.html",
+"Metadata": {
+"AuthorGitHub": "thargy",
+"DateTimeWritten": "09/12/2021 16:00",
+"PreviewImage": "images/blog/dec-2021/structure-chaining.png"
```

Comment:
Do you have the original PSD for this? (happy with the image concept, just want to make it blend a little better!)

Reply:
Sorry, I was being harassed by someone else, so I had to rush it through, not my best work by far.  The original image was off unsplash.com (so no royalty issues), and the edits took me 2 minutes.  Sadly, I won't be able to get back on to my PC tonight.

Reply:
Thankfully, PC was on Sleep and PS was still open so have now sent via DM on Discord.